packages/commonwealth/client/scripts/app.ts

```diff
@@ -928,6 +928,14 @@ Promise.all([$.ready, $.get('/api/domain')]).then(
               scoped: true,
               deferChain: true,
             }),
+            '/:scope/new/contract': importRoute('views/pages/new_contract', {
```

Comment:
we should add unscoped pages for custom domains. see the section above in this file